### PR TITLE
Figures

### PR DESCRIPTION
**Pull request by shak360**
Draft Pull Request for poster/paper figures.

### CODE REVIEW

**Comment by shak360** [comment]
Should be mostly linting now... 

Some things to add on future branches. 

1. figure 3 should be the exponential growth curve of our synthetic lineage method
2. fix analyze method to include multiple tries and choosing the result of the highest likelihood
3. accuracy method should correctly switch states regarding the parameters (currently is hard-coded to switch for only 2 states, fails at one state, or more than two states)
4. figure 12 should be the KL divergence figure
5. adding parallel processing to figure creation
6. clean up code in general

**Comment by shak360** [comment]
@aarmey Sorry again for the gargantuan pull request. It should be the final one in the series of the last changes. The build fails only when fitting fails; everything else should be in order. I've put a list of things to do in the PR and in the issues.

**Comment by shak360** [comment]
Also I agree figure 11 is not done yet. I'm still debating on how to represent AIC. I first wanna see how the choosing the model with the best likelihood does.

File: lineage/figures/figure10.py [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+
+from .figureCommon import getSetup
+from ..Analyze import accuracy, Analyze
+from ..LineageTree import LineageTree
+from ..StateDistribution import StateDistribution
+
+plt.rc('font', **{'family': 'sans-serif', 'size': 25})
```
[the remark this answers is not in the document]
Comment (shak360) [reply lineage/figures/figure10.py]: Is this the same as just moving these lines to `figureCommon.py`?

[the remark this answers is not in the document]
Comment (aarmey) [reply lineage/figures/figure10.py]: Yes, so that they're in that one place rather than multiple.

[the remark this answers is not in the document]
Comment (shak360) [reply lineage/figures/figure10.py]: Yep, seems like it

**Comment by shak360** [comment]
Ok should be done @aarmey. Thanks in advance.

**Pull request merged**